ext/ManifoldsBoundaryValueDiffEqExt.jl

```diff
@@ -1,3 +1,17 @@
+module ManifoldsBoundaryValueDiffEqExt
+
+using Manifolds
+using ManifoldsBase
+
+using Manifolds: affine_connection
+
+import Manifolds: solve_chart_log_bvp, estimate_distance_from_bvp
```

Comment:
It's unfortunately not documented anywhere I know of, but for someone to build a sysimage with Manifolds and Requires, we need to make sure all `using` and `import` statements when using Requires are relative, see https://github.com/JuliaArrays/ArrayInterface.jl/pull/387

Reply:
OK, I will correct it then.